MekHQ/src/mekhq/gui/dialog/PersonnelMarketDialog.java

```diff
@@ -419,7 +419,7 @@ private void filterPersonnel() {
         sorter.setRowFilter(new RowFilter<>() {
             @Override
             public boolean include(Entry<? extends PersonnelTableModel, ? extends Integer> entry) {
-                return nGroup.getFilteredInformation(entry.getModel().getPerson(entry.getIdentifier()));
+                return nGroup.getFilteredInformation(entry.getModel().getPerson(entry.getIdentifier()),hqView.getCampaign().getLocalDate());
```

Comment:
Style change request: spaces between parameters in parameter lists for readability.